src/mmw/apps/modeling/mapshed/tasks.py

```diff
@@ -155,6 +156,11 @@ def collect_data(geop_result, geojson):
     z['SedPhos'] = geop_result['sed_phos']
     z['Area'] = [percent * area * HECTARES_PER_SQM
                  for percent in geop_result['landuse_pcts']]
+
+    # Immediately return an error for AOIs without enough land cover
+    if len(filter(lambda x: x != 0.0, z['Area'])) == 0:
```

Comment:
Stylistic choice, and might be inconsequential since these will never have negative values, but whenever checking that the value is not 0, I prefer `x > 0` to `x != 0`.